recipes/property-manager/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "property-manager" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "19e1f230f95691ae74123fa43c6928d0f0cc464e841d6375941cda4a0585162f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - humanfriendly >=4.12
+    - pip
+    - python
+    - verboselogs >=1.1
```

Comment:
I think you just need ``python`` and ``pip`` on the ``host`` requirements. Let's try! 
:)

Reply:
Thank you very much for your help @marcelotrevisani! This is very helpful especially since I have another Python package to submit. It is another `sonda skeleton pypi` one, so I'll apply your revisions to it as well.

Thanks again!